dataactbroker/README.md

```diff
@@ -1358,26 +1358,46 @@ Successful response will contain the signed S3 URL for the file we're trying to
 
 Invalid certified_files_history_id, requests for a file not related to the submission_id given, or requests for a file that isn't stored in the table will return a 400 error.
 
-#### GET "/v1/list_agencies/"
-Gets all CGACS that the user has submit/certify permissions
+#### GET "/v1/list\_agencies/"
+Gets all CGACs/FRECs that the user has permissions for.
 
-Example input:
-
-None
+##### Sample Request
+`/v1/list_agencies/`
 
-Example output:
+##### Request Params
+N/A
 
-```json
+##### Response (JSON)
+```
 {
     "cgac_agency_list": [
-      {
-        "agency_name": "Sample Agency",
-        "cgac_code": "000"
-      }, ...
+        {
+            "agency_name": "Sample Agency",
+            "cgac_code": "000"
+        },
+        {
+            "agency_name": "Sample Agency 2",
+            "cgac_code": "999"
+        }
+    ],
+    "frec_agency_list": [
+        {
+            "agency_name": "Sample FREC Agency",
+            "frec_code": "0000"
+        }
     ]
 }
 ```
 
+##### Response Attributes
+- `cgac_agency_list`: (list[object]) A list of all cgac agencies (cgac code and agency name) the user has permissions to access.
```

Comment:
I've been switching between `object` and `dict` in the documentation. Which one are we sticking with?

Reply:
As discussed, from now on it's a `dict`